include/AlertRouter.h

```diff
@@ -0,0 +1,13 @@
+#ifndef PROXYSQL_ALERTROUTER_H
+#define PROXYSQL_ALERTROUTER_H
+
+// Acts as a gateway between ProxySQL core and integrations with alerting services.
+// All alerts that need to be pushed to an external service should pass through it.
+// It detects which integrations are enabled and passes the alert to all active
+// integrations.
+class AlertRouter {
```

Comment:
For me it was a bit confusing to have both `AlertRouter` and `AlertServiceConnector` implement a similar interface. Maybe it's worth to explain in 1 sentence the connection between the two.